### ISSUE TITLE
Font size

### ISSUE THREAD

**Issue by marmangarcia** [nopnop2002/Raspberry-ssd1306#13]
Hi, Is there a way to increase the font size?

**Comment by nopnop2002** [comment]
This tool uses two types of fonts.

External font
If you want to use a different font, you need to change this.
https://github.com/nopnop2002/Raspberry-ssd1306/blob/master/oled.c#L163

Internal font
If you want to use a different font, you need to change this..
https://github.com/nopnop2002/Raspberry-ssd1306/blob/master/oled.c#L489

**Comment by marmangarcia** [comment]
yes, I change the filename here https://github.com/nopnop2002/Raspberry-ssd1306/blob/master/oled.c#L163

but its not working, do you have a working samples I want to increase the font size.

**Comment by nopnop2002** [comment]
I don't have working samples.
